internal/commands/inits/data/status_shortcuts.fish

```diff
@@ -22,6 +22,28 @@ function scmpuff_status
     end
 end
 
+function scmpuff_branch
+    scmpuff_clear_vars
+    set -lx scmpuff_env_char "e"
```

Comment:
Clobbering the existing file variable could be problematic: it might not be obvious to a user that if they list branches, the previous file shortcuts that were displayed to them would stop. working. I suspect we need a second variable here (which unfortunately does add to implementation complexity, but I don't know a great way around that).